pkg/services/services.go

```diff
@@ -52,7 +52,10 @@ func GenerateManifest(kubeletArgsFromIgnition map[string]string, vxlanPort strin
 	}
 	// TODO: All payload filenames and checksums must be added here https://issues.redhat.com/browse/WINC-847
 	files := &[]servicescm.FileInfo{}
-	return servicescm.NewData(services, files, cluster.ClusterWideProxyVars)
+	if proxyEnabled {
+		return servicescm.NewData(services, files, cluster.ClusterWideProxyVars)
+	}
+	return servicescm.NewData(services, files, nil)
```

Comment:
Is it possible to remove the need for the `proxyEnabled` parameter?
When proxy is not enabled, cluster.ClusterWideProxyVars should be empty (len=0). Is there a difference in how empty and nil values are treated? I don't think there should be

Reply:
I agree but if that's the case do we even need the proxyenabled parameter in the configmapReconciler struct?